src/custom.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'babel-plugin-relay/macro' {
```

Comment:
what is this module for?

Reply:
This is a file that exports types that Typescript has trouble detecting (usually due to the module itself and not Typescript). [Related issue](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/35707)